tests/test_features/test_pca.py

```diff
@@ -0,0 +1,233 @@
+##########################################################################
+## Imports
+##########################################################################
+
+import unittest
+import numpy as np
+import numpy.testing as npt
+import pytest
```

Comment:
I'm always interested in new packages to make things better, but I think we're not getting any additional benefit in this particular case.